utilities/manual/source/configuration.md

```diff
@@ -797,6 +797,16 @@ The "show all" check box allows to see the complete menu structure of the main w
 Changes to the Slots are discouraged unless you really know what you are doing. No support for changes here is given.
 ```
 
+You can use command `app.loadManagedMenu()` in [script macros](advanced.md#script-macros) to load a file that contains your definitions for menus
```

Comment:
And now it would be nice if the general style of the manual would be followed.
Everything is expressed neutrally, and all of a sudden "you can"...